compiler/rustc_monomorphize/src/collector.rs

```diff
@@ -1351,7 +1351,10 @@ fn create_mono_items_for_default_impls<'tcx>(
                 tcx.def_path_str(item.owner_id.to_def_id())
             );
 
-            if let Some(trait_ref) = tcx.impl_trait_ref(item.owner_id) {
+            if let Some(trait_ref) = tcx
+                .bound_impl_trait_ref(item.owner_id.to_def_id())
+                .map(ty::EarlyBinder::subst_identity)
```

Comment:
think it would be clearer to change this to
```rust
if let Some(trait_ref) = tcx.impl_trait_ref(item.owner_id.to_def_id()) {
    let trait_ref = trait_ref.subst_identity();
}

Reply:
I found a few other places where I did something similar, so I cleaned those up too